src/attributes/utils/data.ts

```diff
@@ -87,6 +93,9 @@ export function getSelectedAttributeValues(
     case AttributeInputTypeEnum.NUMERIC:
       return [attribute.values[0]?.name];
 
+    case AttributeInputTypeEnum.BOOLEAN:
+      return [attribute.values[0]?.boolean || "false"];
```

Comment:
Why is `false` used as string here? This condition will always resolve with either `true` or `"false"`